app/src/main/java/com/techcourse/DispatcherServlet.java

```diff
@@ -40,12 +42,8 @@ protected void service(final HttpServletRequest request, final HttpServletRespon
     }
 
     private void move(final String viewName, final HttpServletRequest request, final HttpServletResponse response) throws Exception {
-        if (viewName.startsWith(JspView.REDIRECT_PREFIX)) {
-            response.sendRedirect(viewName.substring(JspView.REDIRECT_PREFIX.length()));
-            return;
-        }
-
-        final var requestDispatcher = request.getRequestDispatcher(viewName);
-        requestDispatcher.forward(request, response);
+        ModelAndView modelAndView = new ModelAndView(new JspView(viewName));
+        View view = modelAndView.getView();
+        view.render(modelAndView.getModel(), request, response);
```

Comment:
스프링 프레임워크 공식 문서에 뷰 렌더링 조건에 대한 내용이 있어서 가져와봤어요!
```
If a model is returned, the view is rendered. 
If no model is returned, (may be due to a preprocessor or postprocessor intercepting the request, perhaps for security reasons), 
no view is rendered, because the request could already have been fulfilled.
```
번역기를 돌리면
```
모델이 반환되지 않으면(전처리기 또는 후처리기가 보안상의 이유로 요청을 가로채기 때문일 수 있음) 
요청이 이미 완료되었을 수 있으므로 뷰가 렌더링되지 않습니다.
```
라고 하네요!
model이 반환되지 않은 경우 render 메서드를 실행하지 않도록 예외처리를 해 주는 것은 어떨까요?

Reply:
좋아요! 반영했습니다 👍 
뷰 렌더링에 대한 조건은 고려하지 못했는데 좋은 인사이트 감사해요, 레모네!